SwiftCompilerSources/Sources/Optimizer/FunctionPasses/StripObjectHeaders.swift

```diff
@@ -55,6 +55,7 @@ private struct IsBareObjectWalker : ValueDefUseWalker {
          is BranchInst, is CondBranchInst, is SwitchEnumInst,
          is UpcastInst, is UncheckedRefCastInst,
          is BeginDeallocRefInst,
+         is EndInitLetRefInst,
```

Comment:
Thank you so much for adding a different instruction! I like how the name is really clear.